### PR TITLE
[Update] Adapt DSM 2.0

### PR DESCRIPTION
**Pull request by zikunfan**
-  Adapt new MPoW work report and anchor mechanism;
-  Adapt new DSM api
-  Prepping 0.11.0

### CODE REVIEW

File: src/services/swork.ts [hunk 1]
```diff
@@ -31,21 +31,29 @@ export async function reportWorks(
...
+      const rst: [string, number, number] = [
+        '0x' + file.hash,
+        file.size,
+        file.c_block_num,
+      ];
       return rst;
     }),
     req.body['deleted_files'].map((file: any) => {
-      const rst: [string, number] = ['0x' + file.hash, file.size];
+      const rst: [string, number, number] = [
```
Comment (pangwa) [inline src/services/swork.ts:52]: duplicated code with lines above

File: src/services/market.ts [hunk 2]
```diff
@@ -1,74 +1,10 @@
...
-          const sorders = f_map[f_id];
-          if (sorders.length > 0) txRes.order_id = sorders[0];
-        }
-      }
-    }
-  }
-
-  return txRes;
+// Queries
+export async function file(api: ApiPromise, cid: string) {
```
Comment (pangwa) [inline src/services/market.ts:6]: the name looks too common...

File: src/services/swork.ts (hunk 1 above)
Comment (zikunfan) [reply src/services/swork.ts:52]: yes, but 1 is `added`, another is `deleted`

File: src/services/market.ts (hunk 2 above)
Comment (zikunfan) [reply src/services/market.ts:6]: this is align to `chain-side`

**Review by pangwa — APPROVED** [review]

**Pull request merged**